lib/utils.js

```diff
@@ -347,6 +347,32 @@ api.validateCapabilityChain = ({capability, capabilityChain}) => {
   return {valid: true};
 };
 
+/**
+ * A capability to inspect. The capability is compacted into the security
+ * context. Only the required fields are shown here, a capability will contain
+ * additional properties.
+ * @typedef {Object} Capability
+ * @property {string} id - The ID of the capability.
+ * @property {string} invoker - The invoker of the capability.
+ * @property {string} [delegator] - The delegator of the capability. The last
+ *   capability in the chain will not have a delegator.
+ */
+
+/**
+ * The capability to inspect.
+ * @typedef {Object} CapabilityChainDetails
+ * @property {Capability[]} capabilityChain - The capabilities in the chain.
+ * @property {string} invocationTarget - A URL that specifies the ID of the
+ *   root capability.
```

Comment:
```suggestion
 *   invocation target.
```

Reply:
Making change, but Is 'root capability' not accurate?